metadata-models/src/main/pegasus/com/linkedin/metadata/key/MLModelEndpointKey.pdl

```diff
@@ -0,0 +1,40 @@
+namespace com.linkedin.metadata.key
+
+import com.linkedin.common.Urn
+import com.linkedin.common.FabricType
+
+/**
+ * Key for an ML model endpoint
+ */
+@Aspect = {
+  "name": "mlModelEndpointKey"
+}
+record MLModelEndpointKey {
+  /**
+  * Standardized platform urn for the model endpoint
+  */
+  @Searchable = {
+    "fieldType": "URN",
+    "addToFilters": true
+  }
+  platform: Urn
+
+  /**
+  * Name of the MLModelEndpoint
+  */
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  name: string
```

Comment:
where do these names come from?

Reply:
SageMaker – you can set a name for an endpoint